ORBIT/phases/design/_cables.py

```diff
@@ -310,6 +311,72 @@ def _initialize_cables(self):
             name = specs["name"]
             self.cables[name] = Cable(specs)
 
+    def _get_touchdown_distance(self):
+        """
+        Returns the cable touchdown distance measured from the centerpoint of
+        the substructure.
+
+        If depth <= 60, default is 0km (straight down assumed for fixed bottom).
+        If depth > 60, default is 0.3 * depth.
+        """
+
+        _design = f"{self.cable_type}_system_design"
+        depth = self.config["site"]["depth"]
+        touchdown = self.config[_design].get("touchdown_distance", None)
+
+        if touchdown is not None:
+            self.touchdown = touchdown
+
+        else:
+            if depth <= 60:
+                self.touchdown = 0
+
+            else:
+                self.touchdown = depth * 0.3
+
+    @staticmethod
+    def _catenary(a, *data):
+        """Simple catenary equation."""
+
+        d, h = data
+        res = a * np.cosh(h / a) - (d + a)
+        return res
+
+    def _get_catenary_length(self, d, h):
+        """
+        Returns the catenary length of a cable that touches down at depth `d`
+        and horizontal distance `h`.
+
+        Returns
+        -------
+        float
+            Catenary length.
+        """
+
+        a = fsolve(self._catenary, 8, (d, h))
+
+        x = np.linspace(0, h)
+        y = a * np.cosh(x / a) - a
+
+        if not np.isclose(y[-1], d):
+            print(
+                "Warning: Catenary calculation failed. Reverting to simple vertical profile."
+            )
+            return d
+
+        return np.trapz(np.sqrt(1 + np.gradient(y, x) ** 2), x)
+
+    @property
+    def free_cable_length(self):
+        """Returns the length of the vertical portion of a cable section in km."""
+
+        depth = self.config["site"]["depth"]
+
+        if not self.touchdown:
+            return depth / 1000
+
+        return self._get_catenary_length(depth, self.touchdown) / 1000
```

Comment:
This is actually something that we weren't accounting for previously. This is the vertical distance of the cable from substructure to seafloor. You'll see later that it is now excluded from the burial, trenching operations.